apps/site/lib/site_web/controllers/schedule/line_controller.ex

```diff
@@ -257,7 +257,7 @@ defmodule SiteWeb.ScheduleController.LineController do
         group_name: ViewHelpers.mode_name(group),
         routes:
           routes
-          |> Enum.sort_by(&connection_sorter/1)
+          |> Enum.sort_by(& &1.sort_order)
```

Comment:
The affects the connections displayed in the Schedule's sidebar "Connections" accordion.